<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="2">
     <widget class="QSpinBox" name="maxInput">
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="minLabel">
      <property name="text">
       <string>Minimal weight</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="matrixLabel">
      <property name="text">
       <string>Matrix size(n)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="edgesLabel">
      <property name="text">
       <string>Edges count(n)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="stopButton">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="edgesInput">
      <property name="text">
       <string>n</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="QTableWidget" name="tableWidget">
      <property name="rowCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row/>
      <column>
       <property name="text">
        <string>Time(n)</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="maxLabel">
      <property name="text">
       <string>Maximal weight</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="minInput">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="sizeInput">
      <property name="text">
       <string>n</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
